### PR TITLE
Add broadcast receiver to toggle clean status by broadcast

### PR DESCRIPTION
**Pull request by cypressious**

### CODE REVIEW

File: app/src/main/java/com/emmaguy/cleanstatusbar/ToggleReceiver.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package com.emmaguy.cleanstatusbar;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+
+public class ToggleReceiver extends BroadcastReceiver {
+
+    public static final String EXTRA_ENABLED = "enabled";
```
Comment (emmaguy) [inline app/src/main/java/com/emmaguy/cleanstatusbar/ToggleReceiver.java]: Can this be `private` please? Otherwise looks good

Comment (cypressious) [inline app/src/main/java/com/emmaguy/cleanstatusbar/ToggleReceiver.java]: Done!

**Comment by cypressious** [comment]
Note, this will always start the service, if the extra is not specified.

**Comment by emmaguy** [comment]
@cypressious that sounds fine - thank you for this! Will just wait for Travis and merge :tada:

**Comment by cypressious** [comment]
Travis seems happy with the RP

**Pull request merged**